run_tests.sh

```diff
@@ -6,47 +6,48 @@ export TF_CPP_MIN_LOG_LEVEL="3"
 
 # --------------------------------------------------------------------------------------------------------------- TESTS
 
-
 mlFrameworkList=("tensorflow" "scikitlearn")
 for mlFramework in "${mlFrameworkList[@]}"; do
-  echo "Running tests with framework $mlFramework"
-  pytest -q -vv tests/attacks/inference/ --mlFramework=$mlFramework --skip_travis=True --durations=0
+  echo "#######################################################################"
+  echo "############## Running tests with framework $mlFramework ##############"
+  echo "#######################################################################"
+
+  #TODO FIX Failing with Keras at test_membership_inference.test_black_box_keras_loss
+  pytest --cov-report=xml --cov=art --cov-append -q -vv tests/attacks/inference/test_membership_inference.py --mlFramework=$mlFramework --skip_travis=True --durations=0
   if [[ $? -ne 0 ]]; then exit_code=1; echo "Failed attacks/inference tests"; fi
```

Comment:
This TODO has been fixed in the meantime with PR #651 on `dev_1.4.2` and `dev_1.5.0`. Can we now move this run inside of the general loop?

Reply:
ok added it to the main loop